### DISCUSSION TITLE
Data Importer: How to update a link path?

### DISCUSSION THREAD

**Discussion by kopieMK** [pimcore/pimcore#13235] [Q&A / Support]
Hi guys!

I am trying to update some data objects. The import works for things like a simple text field (e.g. name) ... but now I want to update the path inside a link. And I can't find the right mapping here. 

In my importer configuration under the Data Hub in the "Import Settings" tab and there in the "Mappings" tab in the "Data Targets" (Type: Direct) section, I can't find the "link" field in the "Field name" dropdown. This is a field of type "link" in my product class. (see screenshot)

![data-importer_mapping-link](https://user-images.githubusercontent.com/50910488/191924809-c8d78969-2d33-4d01-a76d-9e68dc42157e.jpg)

For better understanding, when I export the data, I can select the Link field and extract the path using 2 nested "Operator Any Getter" functions. But now how can I import the modified path again?
I use a CSV for the import with the data like in the following screenshot:

![data-importer_csv-data](https://user-images.githubusercontent.com/50910488/191924828-8dd22381-90d5-4f14-b567-2505b43d44e8.jpg)

Does anyone have experience with this or a first idea?

Greetings

Magnus

**Comment by robertSt7** [comment]
Hello @kopieMK I am not sure, but maybe you can try to achieve it with some of the events https://pimcore.com/docs/platform/Data_Importer/Extending/Events
